website/src/queries-init.js

```diff
@@ -166,7 +166,15 @@ function getQueryOrDefault(defaultVal) {
   if (!JQ_REPO_FIELD.val()) {
     JQ_REPO_FIELD.val(defaultVal);
   }
-  return JQ_REPO_FIELD.val();
```

Comment:
These changes to `getQueryOrDefault` actually belong in `queries-logic.js` : please move them over there.
This method should remain "dumb" and only return the content of the input field.